api/app/tests/common.py

```diff
@@ -206,3 +206,8 @@ def default_mock_requests_session_post(self, url, data=None, json=None, **kwargs
     """ Return a mocked request response from a request.Session object """
     return default_mock_requests_post(url, data, json, **kwargs)
 # pylint: enable=redefined-outer-name
+
+
+def str_to_bool(value: str):
```

Comment:
Already exists in stdlib: https://docs.python.org/3/distutils/apiref.html#distutils.util.strtobool

Reply:
Sweet! Thanks - didn't know what had that.